compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeLayer.desktop.kt

```diff
@@ -452,11 +452,18 @@ private fun ComposeScene.onMouseWheelEvent(
 
 @OptIn(ExperimentalComposeUiApi::class)
 private val MouseEvent.buttons get() = PointerButtons(
-    isPrimaryPressed = (modifiersEx and MouseEvent.BUTTON1_DOWN_MASK) != 0 && !isMacOsCtrlClick,
-    isSecondaryPressed = (modifiersEx and MouseEvent.BUTTON3_DOWN_MASK) != 0 || isMacOsCtrlClick,
-    isTertiaryPressed = (modifiersEx and MouseEvent.BUTTON2_DOWN_MASK) != 0,
-    isBackPressed = (modifiersEx and MouseEvent.getMaskForButton(4)) != 0,
-    isForwardPressed = (modifiersEx and MouseEvent.getMaskForButton(5)) != 0,
+    isPrimaryPressed = ((modifiersEx and MouseEvent.BUTTON1_DOWN_MASK) != 0
```

Comment:
Can you add a similar comment for this workaround, as we done in skiko code?

Reply:
Done.